README.md

```diff
@@ -50,6 +50,7 @@ All OpenTelemetry libraries are distributed via packagist, notably:
 - Semantic Conventions: [open-telemetry/sem-conv](https://packagist.org/packages/open-telemetry/sem-conv)
 - Context: [open-telemetry/context](https://packagist.org/packages/open-telemetry/context)
 - Contrib: [open-telemetry/sdk-contrib](https://packagist.org/packages/open-telemetry/sdk-contrib)
+- Extensions: [open-telemetry/extension-*](https://packagist.org/search/?query=open-telemetry%2Fextension)
```

Comment:
https://packagist.org/search/?query=open-telemetry&tags=extension instead (which uses the `extension` tag that we added) ?